bounties_api/analytics/management/commands/timeline_generator.py

```diff
@@ -233,12 +241,12 @@ def generate_timeline(time_frame, schema):
 
 class Command(BaseCommand):
     def handle(self, *args, **options):
-
+        BountiesTimeline.objects.all().delete()
```

Comment:
Why are you deleting these records!?

Reply:
:/

Reply:
@villanuevawill oh shit, my bad, that was for testing purposes, was faster than to recreate the db, forgot to delete

Reply:
and it's gone